app/uk/gov/hmrc/incorporatedentityidentification/testonly/GetCtReferenceStubController.scala

```diff
@@ -111,7 +111,7 @@ class GetCtReferenceStubController @Inject() (controllerComponents: ControllerCo
     "9870565"  -> "7161525510"
   )
 
-  private val pillarTwoTestData: Map[String, String] = Map(
+  private val atsTestData: Map[String, String] = Map(
```

Comment:
The entries for CRN values of 77442946 and 77431892 seem to be missing and there are duplicate entries for the CRN 12345610.